test/test-sock-send-recv.c

```diff
@@ -0,0 +1,162 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+#include "uv.h"
+#include "uvwasi.h"
+#include "wasi_serdes.h"
+#include <stdio.h>
+
+#define PREOPEN_SOCK 3
+#define INVALID_SOCK 42
+#define DEFAULT_BACKLOG 5
+
+int delayedThreadTime =  5000;
```

Comment:
Prefer snake case over camelCase.

```suggestion
static int delayed_thread_time = 5000;
```